### DISCUSSION TITLE
Media Manager - Additional Data - Popup - Improve labels on screen

### DISCUSSION THREAD

**Discussion by ChristineWk** [joomla/joomla-cms#47070] [2. Cold Features]
### Steps to reproduce the issue

see previous (part of Issue #38059 - will be closed afterwards)

The fields are unusable.

![screen shot 2022-06-19 at 16 06 11](https://issues.joomla.org/uploads/1/0b24e559452836cc1ebec04606f48f24.png)

### Expected result
To make it editable on a mobile.

### Actual result

Why aren't the labels just placed discreetly above the fields, but fill up half the width?

### Additional comments
https://github.com/joomla/joomla-cms/issues/38059#issuecomment-1156214940

**Comment by brianteeman** [comment]
As the screenshots are not available anymore its impossible to determine what the issue is here

**Comment by ChristineWk** [comment]
![additionaldata](https://github.com/user-attachments/assets/6a885272-3d12-46ba-8d89-36588e2e11ed)
![additionaldata-german](https://github.com/user-attachments/assets/ce19d862-96de-4a0c-92ed-f27ac8bc143b)
